pkgs/build-support/trivial-builders.nix

```diff
@@ -63,6 +63,9 @@ rec {
         # Pointless to do this on a remote machine.
         preferLocalBuild = true;
         allowSubstitutes = false;
+        meta = {
+          binPath = destination;
```

Comment:
maybe only for `executable`s?

Reply:
While working on this PR I discovered that there is already an `executables` (plural) attribute. I have no idea what it's for.
```
$ rg "executables = \["
pkgs/os-specific/linux/autofs/default.nix
45:    executables = [ "automount" ];

pkgs/tools/text/patchutils/generic.nix
23:    executables = [ "combinediff" "dehtmldiff" "editdiff" "espdiff"

pkgs/tools/backup/dirvish/default.nix
13:  executables = [ "dirvish" "dirvish-runall" "dirvish-expire" "dirvish-locate" ];
```

Reply:
summoning @winpat who added dirvish

Reply:
quick review suggests those are used only for build process, so probably not a subject for this PR

Reply:
@zimbatm What @danbst said.